src/org/opendatakit/briefcase/util/ServerFetcher.java

```diff
@@ -262,13 +268,13 @@ public SubmissionDownloadChunk(List<String> uriList, String websafeCursorString)
     }
   };
 
-  private boolean downloadAllSubmissionsForForm(File formInstancesDir, DatabaseUtils formDatabase, BriefcaseFormDefinition lfd,
-      FormStatus fs) {
-    boolean allSuccessful = true;
+  private TerminationStatus downloadAllSubmissionsForForm(File formInstancesDir, DatabaseUtils formDatabase, BriefcaseFormDefinition lfd,
+                                                    FormStatus fs) {
 
     RemoteFormDefinition fd = (RemoteFormDefinition) fs.getFormDefinition();
 
     int count = 1;
+    int filesSkipped = 0;
```

Comment:
Is there any reason this can't still be a boolean (since we're only checking for zero/non-zero)?